packages/forms-web-app/src/controllers/appeals/your-appeals.js

```diff
@@ -11,7 +16,12 @@ exports.get = async (req, res) => {
 		let appeals = await apiClient.getUserAppealsById(user.id);
```

Comment:
FYI this returns `(AppealCase|AppealSubmission)[]` but the utility/dashboard functions don't seem to expect that.

Reply:
We have sought to update to reflect this